client/ayon_core/tools/push_to_project/ui/window.py

```diff
@@ -14,12 +14,16 @@
 )
 
 
-class PushToContextSelectWindow(QtWidgets.QWidget):
-    def __init__(self, controller=None):
+class PushToContextSelectWindow(QtWidgets.QDialog):
+    def __init__(
+        self, controller=None, library_filter=True, context_only=False
```

Comment:
Both filters should be defined on controller and not on window.

Reply:
But they are only needed in in the window, since that is where the code is. Have you got a suggestion for separating the code?